File: solutions2015/PRA3/daniel.wu.14@gmail.com.py
__author__ = 'Daniel Wu'

def __main__():
    index = 0
    num_pot = int(input())
    pots = []
    for n in range(num_pot):
        pots.append(int(input()))
    for k in range(num_pot):
        left_sum = 0
        right_sum = 0
        for i in range(k):
            left_sum += pots[i]
        for l in range(num_pot-1,k,-1):
            right_sum += pots[l]

        if left_sum == right_sum and left_sum != 0:
            index = k

    print(index)

if __name__ == '__main__':
    __main__()

File: solutions2015/PRA3/daniel.wu.14@gmail.com.java
import java.util.*;
public class PRA3 {
	public static void main(String[] args){
		int index = 0;
		Scanner io = new Scanner(System.in);
		int numPot = io.nextInt();
		int[] potionList = new int[numPot];
		for(int i = 0; i < numPot; i++){
			potionList[i] = io.nextInt();
		}
		for(int i = 0; i < numPot; i++){
			int leftSum = 0;
			int rightSum = 0;
			for(int j = 0; j < i; j++){
				leftSum += potionList[j];
				
			}
			for(int k = numPot-1; k > i; k--){
				rightSum += potionList[k];
			}
			if(leftSum == rightSum && leftSum != 0){ 
				index = i;
				break;
			}
			leftSum = 0;
			rightSum = 0;
		}
		System.out.println(index);
	}
}
